src/lib/github/addIssuesToRepo.ts

```diff
@@ -11,21 +12,38 @@ export default async function addIssuesToRepo (context: Context, newIssues: Issu
   const octokit = context.github;
   const { owner, repo } = getBasicRepoProps (context);
   const ghIssues = await getAllRepoIssues(context);
+
   newIssues.forEach(async ({title, body, authors}) => {
     const assignees = uniq(authors);
+
     const currentGHIssue = find(ghIssue => ghIssue.title === title, ghIssues);
+    const currentCommentBody = stringifyCommentsArray(currentGHIssue.comments);
+    const newGHIssueComments = !(currentCommentBody === body)
+     ? newIssueBody(currentCommentBody, body)
+     : currentCommentBody;
+
     const fields = {
       owner,
       repo,
-      body,
+      body: newGHIssueComments,
       title,
       labels: ["GH-TODO-BOT"],
       assignees,
       ...(currentGHIssue ? { number: currentGHIssue.number } : {})
     };
+
     return currentGHIssue
       ? octokit.issues.createComment(fields)
       : octokit.issues.create(fields);
   });
 
 }
+
+function stringifyCommentsArray (commentsArray: string[]): string {
+  return commentsArray.join("\n");
+}
+
+function newIssueBody(currentIssueBody: string, newIssueBody: string): string {
+  const regex = /currentIssueBody/gi;
```

Comment:
That is not how regex variables are created. see https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp